### PR TITLE
Editing getting started

### PR DESCRIPTION
**Pull request by Buzzardo**
I made the document more formal and corrected sentence errors.

### CODE REVIEW

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -65,51 +65,51 @@ You can also deploy Spring Boot applications to any Servlet 3.0+ compatible cont
...
 [[getting-started-installation-instructions-for-java]]
-=== Installation instructions for the Java developer
-You can use Spring Boot in the same way as any standard Java library. Simply include the
+=== Installation Instructions for the Java Developer
+You can use Spring Boot in the same way as any standard Java library. To do so, include the
 appropriate `+spring-boot-*.jar+` files on your classpath. Spring Boot does not require
-any special tools integration, so you can use any IDE or text editor; and there is
-nothing special about a Spring Boot application, so you can run and debug as you would
+any special tools integration, so you can use any IDE or text editor. Also, there is
+nothing special about a Spring Boot application, so you can run and debug a Spring Boot program as you would
```
Comment (wilkinsona) [inline spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: The switch from "application" to "program" reads a little strangely to me. For consistency, shouldn't it be "application" again?

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc [hunk 2]
```diff
@@ -251,17 +251,17 @@ endif::[]
 
 [[getting-started-installing-the-cli]]
 === Installing the Spring Boot CLI
-The Spring Boot CLI is a command line tool that can be used if you want to quickly
-prototype with Spring. It allows you to run http://groovy-lang.org/[Groovy] scripts,
-which means that you have a familiar Java-like syntax, without so much boilerplate code.
+The Spring Boot CLI (Command Line Interface) is a command line tool that you can use to quickly
```
Comment (wilkinsona) [inline spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: What's the policy for expanding CLI to Command Line Interface? It was already done [above](https://github.com/spring-projects/spring-boot/pull/10763/files#diff-5c18fc93d7063a97159926b2a132dc13R77) but then isn't done below. I'm just curious so that we can follow the convention in the future.

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc [hunk 3]
```diff
@@ -358,14 +358,14 @@ install the Spring Boot CLI is:
...
+The Spring Boot CLI includes scripts that provide command completion for
+the http://en.wikipedia.org/wiki/Bash_%28Unix_shell%29[BASH] and
 http://en.wikipedia.org/wiki/Zsh[zsh] shells. You can `source` the script (also named
-`spring`) in any shell, or put it in your personal or system-wide bash completion
-initialization. On a Debian system the system-wide scripts are in `/shell-completion/bash`
+`spring`) in any shell or put it in your personal or system-wide bash completion
+initialization. On a Debian system, the system-wide scripts are in `/shell-completion/bash`
 and all scripts in that directory are executed when a new shell starts. To run the script
-manually, e.g. if you have installed using SDKMAN!
+manually if you have installed using SDKMAN!, use the following commands:
```
Comment (wilkinsona) [inline spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: I think SDKMAN! needs to be an example here. There are others reasons why you may want to install the script manually, not just if you installed the CLI with SDKMAN!

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc [hunk 4]
```diff
@@ -602,64 +601,64 @@ file named `src/main/java/Example.java`:
...
 [[getting-started-first-application-run]]
-=== Running the example
-At this point our application should work. Since we have used the
-`spring-boot-starter-parent` POM we have a useful `run` goal that we can use to start
+=== Running the Example
+At this point, your application should work. Since you used the
+`spring-boot-starter-parent` POM, you have a useful `run` goal that you can use to start
 the application. Type `mvn spring-boot:run` from the root project directory to start the
-application:
+application. You should output similar to the following:
```
Comment (wilkinsona) [inline spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: Missing "see"? You should output -> You should see output

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc [hunk 5]
```diff
@@ -678,42 +677,42 @@ application:
...
-Java does not provide any standard way to load nested jar files (i.e. jar files that are
+Java does not provide a standard way to load nested jar files (jar files that are
 themselves contained within a jar). This can be problematic if you are looking to
 distribute a self-contained application.
 
-To solve this problem, many developers use "`uber`" jars. An uber jar simply packages
-all classes, from all jars, into a single archive. The problem with this approach is that
-it becomes hard to see which libraries you are actually using in your application. It can
+To solve this problem, many developers use "`uber`" jars. An uber jar packages
+all the classes from all the contained jars into a single archive. The problem with this approach is that
```
Comment (wilkinsona) [inline spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: Contained isn't quite right here. Perhaps just "all the jars" or, more accurately, "all the application's dependencies' jars"

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc (hunk 1 above)
Comment (Buzzardo) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: "program" matches "any other Java program". That's the selling point in the sentence, so I thought it best to match "program".

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc (hunk 2 above)
Comment (Buzzardo) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: The policy is to define it the first time it's used and then periodically throughout the document. I did define it at the first instance in the document (currently line 77). Given that the term appears in the heading here, I thought defining it here too would be a good idea.

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc (hunk 3 above)
Comment (Buzzardo) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: OK. I made that change.

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc (hunk 4 above)
Comment (Buzzardo) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: Oops. That's why I need reviewers as much as anyone else does. Thanks for catching it.

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc (hunk 5 above)
Comment (Buzzardo) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: I find possessives of possessives to be hard to follow, so I changed the sentence to "An uber jar packages all the classes from all the application's dependencies into a single archive."

Does that work for you?

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc (hunk 1 above)
Comment (wilkinsona) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: Got you. I still think it should be "application". A Spring Boot application is a particular type of Java program. I haven't heard of anyone refer to a Spring Boot program before. Application also ties in with the `@SpringBootApplication` annotation that pretty much every Boot app uses.

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc (hunk 2 above)
Comment (wilkinsona) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: Thanks

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc (hunk 1 above)
Comment (Buzzardo) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: OK. I'll put it back to application.
